fe/fe-core/src/main/java/org/apache/doris/backup/BackupJob.java

```diff
@@ -358,6 +358,18 @@ private void prepareAndSendSnapshotTask() {
         taskProgress.clear();
         taskErrMsg.clear();
         AgentBatchTask batchTask = new AgentBatchTask();
+
+        // remove table which has no partition
+        tableRefs = tableRefs.stream().filter(tableRef -> {
+            String tblName = tableRef.getName().getTbl();
+            Table tbl = db.getTableNullable(tblName);
+            if (tbl != null && tbl.getType() == TableType.OLAP) {
+                OlapTable olapTable = (OlapTable) tbl;
+                return !olapTable.getAllPartitions().isEmpty();
```

Comment:
What kind of table does not have a partition？

Reply:
user creates a table like #8886, not create partition or  drops the partition.

Reply:
@yangzhg are there any questions about table without partition？